websites/docs-smoke-test/src/content/components/rss-feeds.mdx

```diff
@@ -11,12 +11,13 @@ import { RssFeeds } from '@commercetools-docs/ui-kit'
 ## Table with one feed
 
 <RssFeeds dataSources={[
-    "/docs-smoke-test/releases/feed.xml"
+"/docs-smoke-test/releases/feed.xml"
 ]}/>
 
 ## Table with multiple feeds
 
 <RssFeeds dataSources={[
-    "/docs-smoke-test/releases/feed.xml",
-    "/api-docs-smoke-test/releases/feed.xml"
-]}/>
\ No newline at end of file
+"/docs-smoke-test/releases/feed.xml",
+"/api-docs-smoke-test/releases/feed.xml",
+"/self-learning-smoke-test/releases/feed.xml"
```

Comment:
Adding a new feed with only one release note (with a new date) to create the use case in the smoke test.